### PR TITLE
DBpedia composers extraction

### PR DESCRIPTION
**Pull request by angelinrashmi2000**
Server code is removed
Dummy values are removed and replaced by null
Included jsesc so that the request for URL with special characters gets executed successfully
Increased the sleep from 300 to 1000 as some of the responses are not processed 
Extracted placeofDeath, tags, release, instrument etc

### CODE REVIEW

**Review by ShilpaGhanashyamGore — APPROVED** [review]
You could add some comments explaining why certain actions are done Eg:ReplaceURLandUnderscore() from DBPedia p.o.v.

File: Scrap_Dbpedia_Composers/dbPedia_17thCentComposers.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+function replaceURLAndUnderscore(str) {
+    str = str.replace("http://dbpedia.org/resource/", "")
+    return replaceAll(str, "_", " ")
+}
+
+function getResults(queryURL, callback) {
+    name = "done";
+    request(jsesc(queryURL), function(error, response, html) {
+        console.log(queryURL);
+        if (html) {
```
Comment (kordianbruck) [inline Scrap_Dbpedia_Composers/dbPedia_17thCentComposers.js]: Inverting the `if` will save you indentation. 
```
if(!html) {
   return;
}
```

File: Scrap_Dbpedia_Composers/dbPedia_17thCentComposers.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    source_link: queryURL
+                });
+
+            }
+        }
+    });
+    return callback(name);
+}
+
+const url = 'http://dbpedia.org/page/Category:17th-century_classical_composers';
```
Comment (kordianbruck) [inline Scrap_Dbpedia_Composers/dbPedia_17thCentComposers.js]: Please put constants at the top of a file

File: Scrap_Dbpedia_Composers/dbPedia_18thCentComposers.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
+const http = require('http');
```
Comment (kordianbruck) [inline Scrap_Dbpedia_Composers/dbPedia_18thCentComposers.js:1]: Why is it necessary to have separate files for each century? Does the structure differ too much from each other?

**Comment by kordianbruck** [comment]
@angelinrashmi2000 please delete the branch after merging it! Just press that button below.

**Pull request merged**